#! /bin/sh /usr/share/dpatch/dpatch-run
## arm_build.patch.dpatch by  <lool@babbage>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes build on arm; see Debian #489175

@DPATCH@
diff -urNad kexec-tools-20080324~/kexec/arch/arm/kexec-zImage-arm.c kexec-tools-20080324/kexec/arch/arm/kexec-zImage-arm.c
--- kexec-tools-20080324~/kexec/arch/arm/kexec-zImage-arm.c	2008-03-21 03:13:12.000000000 +0000
+++ kexec-tools-20080324/kexec/arch/arm/kexec-zImage-arm.c	2009-01-31 18:30:31.000000000 +0000
@@ -11,7 +11,8 @@
 #include <stdint.h>
 #include <getopt.h>
 #include <arch/options.h>
+#include <unistd.h>
 #include "../../kexec.h"
 
 #define COMMAND_LINE_SIZE 1024
